### ISSUE TITLE
[BUG] Unable to build Supply Ship. Boat tag is missing from recipe. 

### ISSUE THREAD

**Issue by Caprikey** [ldtteam/minecolonies#9404]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Verison?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [X] Single Player
- [X] Multi Player

### Minecraft Version

1.20.1

### MineColonies Version

1.1.160-BETA

### Structurize Version

1.0.633-BETA

### Related Mods and their Versions

Playing: All the Mods 9-0.1.6 through Curseforge

Forge: 47.1.3
BlockUI: 0.0.94-ALPHA
Domum Ornamentum: 1.0.129-BETA

### Current Behavior

When attempting to craft the Supply Ship, it does not work. JEI has the item but the recipe displays five red squares with diagonal lines. When hovering over the recipe materials, the error that shows for the crafting material item is: "Empty Tag: minecraft:boats". Pressing the add to craft table generates missing items error. Boats are able to be crafted without issue. Boats arranged on the crafting table/crafting table on a stick do not shows as a valid craftable item, i.e. Empty Results Box.  Tested with oak and birch boats. Issue has been present since "All the Mods 9 - 0.1.2". Supply camp works. 
![MineColonies-SupplyShip-Error](https://github.com/ldtteam/minecolonies/assets/5361497/a39cbdce-407b-45cc-bcf8-0e22d7c5aae5)
![MineColonies-SupplyShip-Error2](https://github.com/ldtteam/minecolonies/assets/5361497/56f5844b-b736-4893-a360-34e26c244f9b)
![MineColonies-SupplyShip-Error3](https://github.com/ldtteam/minecolonies/assets/5361497/217817d8-b296-4a5f-88a2-19eaf472cd6f)

### Expected Behavior

To be able to craft the supply ship. 

### Reproduction Steps

1. Launch All The Mods 9
2. Collect necessary supplies for Crafting Table
3. Craft Crafting Table
4. Collect necessary supplies for 5 Boats
5. Craft 5 Boats
6. Attempt to Craft Supply Ship

### Logs

https://gist.github.com/Caprikey/b39e34e975cf5db52bab5fd96249f5a3

### Anything else?

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by MotionlessTrain** [comment]
The boat tag is a vanilla tag. The fact that it misses means something is broken in your modpack

**Comment by Caprikey** [comment]
> The boat tag is a vanilla tag. The fact that it misses means something is broken in your modpack

Are you recommending that I submit this bug on the "All The Mods 9" github? If so, I apologize for the confusion on my end and will do so. 

Otherwise, As far as my configuration goes... I'm using the Curseforge application to download and install the All The Mods 9  modpack. This occurs on two separate computers with two different accounts running two different Windows OS.

**Comment by MotionlessTrain** [comment]
> Are you recommending that I submit this bug on the "All The Mods 9" github? If so, I apologize for the confusion on my end and will do so.

I am, yes. This looks to be a bug on their side

**Comment by Caprikey** [comment]
It was most definitely an issue on the modpack's end. I have found a potential solution, not sure if I've broken other things in their modpack but it works now for me hahaha. 

In their KubeJS directory, under server_scripts, in the tags.js file, they have a hardcoded temporary fix coded for BambooEverthing Mod breaking the boats tag until version 2.2.4. This is because in version 2.3.0, Bamboo Everything removed the raft causing the issue, and the current build of BambooEverything is 3.0.3. 

I commented out lines 81-97 and the recipe now works.

Updating this thread for any one else who may come hunting for a possible solution. 

My thread on All The Mods 9 is located at https://github.com/AllTheMods/ATM-9/issues/442
